Source/WebKit/UIProcess/ios/forms/WKDatePickerPopoverController.mm

```diff
@@ -86,6 +86,12 @@ - (instancetype)initWithDatePicker:(UIDatePicker *)datePicker
     _contentSize.width = 2 * marginSize + std::max<CGFloat>(datePickerSize.width, accessoryViewSize.width);
     _contentSize.height = toolbarBottomMargin + 2 * marginSize + datePickerSize.height + accessoryViewSize.height;
 
+#if PLATFORM(VISION)
```

Comment:
I wonder if this really needs to be an idiom check?

Reply:
Good question! I will try a iOS app.

Reply:
You're right. Also, it follows the app idiom, not the web view's.